File: Part4/Python/Function_IncorrectFunctionName.py
def main():
    print( "Type 1 and press enter." )
    user_input = input()
    
    # Function name is wrong - should be int()
    entered_integer = string_to_number( user_input )
    
    print( f"The user entered the integer { entered_integer }" )

if __name__ == "__main__":
    main()

File: Part4/Fortran/Function_IncorrectFunctionName.f90
program function_incorrect_function_name
    implicit none
    character( len=100 ) :: user_input
    integer :: entered_integer
    
    write( *, * ) 'Type 1 and press enter.'
    read( *, '(A)' ) user_input
    
    ! Function name is wrong - should use internal read
    entered_integer = string_to_number( user_input )
    
    write( *, * ) 'The user entered the integer ', entered_integer
    
end program function_incorrect_function_name
